zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -85,6 +86,7 @@
   public static Server jettyWebServer;
   public static NotebookServer notebookWsServer;
   public static Helium helium;
+  public static UsersRepo usersRepo;
```

Comment:
I would sugeest not to think too far, just use a proper name to indicate the current purpose. We can refactor that in future if necessary.  It just like `TODO` in codebase that unfortunately many people will never do the `TODO`. @tinkoff-dwh What do you think ?